### PR TITLE
Enhancements UI and navigation part 2

### PR DESCRIPTION
**Pull request by ramirobg94**

### CODE REVIEW

File: src/screens/Form/components/EntityPage/components/TableEntity/components/Row/index.js [hunk 1]
```diff
@@ -23,8 +38,28 @@ const Row = ({item: parentNode, headers, getWidth, onPress}) => {
...
+      title: t('Form:deleteNode.alert.title'),
+      message: t('Form:deleteNode.alert.message', {
+        name: `${parentLabel} [${parentEntityKeyString}]`,
+      }),
+      acceptText: t('Form:deleteNode.alert.accept'),
+      dismissText: t('Form:deleteNode.alert.dismiss'),
+      onAccept: () => {
+        dispatch(formActions.deleteNodeEntity({node: parentNode}));
+      },
+      onDismiss: () => null,
```
Comment (SteRiccio) [inline src/screens/Form/components/EntityPage/components/TableEntity/components/Row/index.js]: can this onDismiss function be optional?

File: src/screens/Form/components/EntityPage/components/common/EntityPanel/Navigation/component.js [hunk 2]
```diff
@@ -386,11 +387,19 @@ const Next = ({parent}) => {
     formSelectors.getParentEntityNodeDef,
   );
   const isEntityShowAsTable = useSelector(formSelectors.isEntityShowAsTable);
-
+  const isTable = useIsTable();
+  const isMultiple = useMemo(
+    () => NodeDefs.isMultiple(currentEntityNodeDef),
+    [currentEntityNodeDef],
+  );
```
Comment (SteRiccio) [inline src/screens/Form/components/EntityPage/components/common/EntityPanel/Navigation/component.js]: I don't think you need to use useMemo in this case, "isMultiple" is just a primitive (boolean)

**Review by SteRiccio — APPROVED** [review]
approved with some minor comments

File: src/screens/Form/components/EntityPage/components/common/EntityPanel/Navigation/component.js (hunk 2 above)
Comment (ramirobg94) [reply src/screens/Form/components/EntityPage/components/common/EntityPanel/Navigation/component.js]: solved

File: src/screens/Form/components/EntityPage/components/TableEntity/components/Row/index.js (hunk 1 above)
Comment (ramirobg94) [reply src/screens/Form/components/EntityPage/components/TableEntity/components/Row/index.js]: solved

**Pull request merged**